core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -128,8 +126,7 @@ static void tee_ltc_reg_algs(void)
 }
 
 
-#if defined(_CFG_CRYPTO_WITH_HASH) || defined(CFG_CRYPTO_RSA) || \
-	defined(CFG_CRYPTO_HMAC)
+#if defined(CFG_CRYPTO_RSA) || defined(CFG_CRYPTO_DSA)
```

Comment:
I don't get it. IMO that should be `#if defined(CFG_CRYPTO_RSA)` and `case TEE_ALG_DSA_SHA1:` at line 146 should be removed.

Reply:
I'm sorry, it seems I forgot to push the last patch.

Reply:
No problem. Looks OK now.